### PR TITLE
Bump xdr

### PR DESCRIPTION
**Pull request by sisuresh**

### CODE REVIEW

**Comment by sisuresh** [comment]
@leighmcculloch what should we do about the `StellarCoreFuturenet` formula? Should we delete it or update/rename it to just reflect latest stellar-core. I don't think anyone is using it, so maybe deleting makes sense?

File: Formula/stellar-xdr.rb [hunk 1]
```diff
@@ -11,6 +11,6 @@ def install
   end
 
   test do
-    assert_match "stellar-xdr 0.0.17", shell_output("#{bin}/stellar-xdr version")
+    assert_match "stellar-xdr v20.0.0-rc1", shell_output("#{bin}/stellar-xdr version")
```
Comment (leighmcculloch) [inline Formula/stellar-xdr.rb]:
```suggestion
    assert_match "stellar-xdr 20.0.0-rc1", shell_output("#{bin}/stellar-xdr version")
```

**Review by leighmcculloch — CHANGES_REQUESTED** [review]

**Comment by leighmcculloch** [comment]
> @leighmcculloch what should we do about the `StellarCoreFuturenet` formula? Should we delete it or update/rename it to just reflect latest stellar-core. I don't think anyone is using it, so maybe deleting makes sense?

We can delete it.

**Comment by sisuresh** [comment]
> > @leighmcculloch what should we do about the `StellarCoreFuturenet` formula? Should we delete it or update/rename it to just reflect latest stellar-core. I don't think anyone is using it, so maybe deleting makes sense?
> 
> We can delete it.

Deleted. Thanks.

**Review by leighmcculloch — APPROVED** [review]

**Pull request merged**